content/articles/account-securing.md

```diff
@@ -5,13 +5,40 @@ categories:
 - Account
 ---
 
-# Securing your DNSimple Account
+# Securing Your DNSimple Account
 
-Some simple ways to further secure your DNSimple account:
+### Table of Contents {#toc}
 
-- Use a password for your account that includes letters, numbers and punctuation. You can even use multiple words or made up phrases.
-- Use a long password (16 characters or more).
-- Never give your password to someone you don't know. This includes DNSimple support - we'll never ask for your password to help troubleshoot issues.
-- Use a unique password. Don't reuse a password from another service.
-- Don't share your account with others. You can [share the domain](/articles/sharing-domain) or [use the Teams feature](/articles/account-users) to allow other users to access your domains.
-- For an extra level of security, [enable multi-factor authentication](/articles/multi-factor-authentication).
+* TOC
+{:toc}
+
+---
+
+Domains are some of the most important assets for any organization, business, or personal project. We do everything we can to keep your domains and information secure, but there are steps you can take to further secure your DNSimple account.
+
+<alert> 
+If you've noticed fraudulent activity, or suspect someone has taken over your account, contact security@dnsimple.com. 
```

Comment:
This is from the security policy, but lmk if that's not right

Reply:
Customers will most likely have eyes on their account faster if they write to support@dnsimple.com. We investigate and then we internally flag for security/compliance review.

Reply:
Agree with Stephanie.

Reply:
i have updated this ^^